bindings/python/vara-feature/pybind_FeatureSourceRange.cpp

```diff
@@ -49,7 +49,12 @@ void init_feature_location_module(py::module &M) {
       .def_property_readonly(
           "end", &vf::FeatureSourceRange::getEnd,
           R"pbdoc(Get the end `LineColumnOffset` of this `Location`.)pbdoc")
-      .def("__str__", &vf::FeatureSourceRange::toString);
+      .def("__str__", &vf::FeatureSourceRange::toString)
+      .def("__eq__", [](const vf::FeatureSourceRange &Self,
+                        const vf::FeatureSourceRange *Other) {
+        if (!Other) return false;
```

Comment:
```suggestion
        if (!Other) {
            return false;
        }
```
please always use brackets, a if without is a bug ready to happen